extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -288,23 +288,23 @@
 # @since 2.1 enabled default types, to disable the functionality either set the
 # variable to array() or false
 ##
-$GLOBALS['smwgSearchByPropertyFuzzy'] = array( '_num', '_txt', '_dat' );
+$smwDefaults['smwgSearchByPropertyFuzzy'] = array( '_num', '_txt', '_dat' );
 ##
 
 ###
 # Number results shown in the listings on pages in the namespaces Property,
 # Type, and Concept. If a value of 0 is given, the respective listings are
 # hidden completely.
 ##
-$GLOBALS['smwgTypePagingLimit'] = 200;    // same number as for categories
-$GLOBALS['smwgConceptPagingLimit'] = 200; // same number as for categories
-$GLOBALS['smwgPropertyPagingLimit'] = 25; // use smaller value since property lists need more space
+$smwDefaults['smwgTypePagingLimit'] = 200;    // same number as for categories
+$smwDefaults['smwgConceptPagingLimit'] = 200; // same number as for categories
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)